src/frameworks/foundation/ns_set.rs

```diff
@@ -83,6 +84,12 @@ pub const CLASSES: ClassExports = objc_classes! {
 // TODO: more init methods, etc
 
 // TODO: accessors
+- (id)allObjects {
+    // TODO: actually implement this
+    let object = env.objc.borrow_mut::<SetHostObject>(this).dict.iter_keys().next().unwrap();
```

Comment:
This looks very close to an actual implementation. I think you would just need to use `.collect()` rather than `.next().unwrap()`.

It wouldn't work with foreign subclasses of NSSet but that's fine I guess.

Reply:
Should be fixed now, I rewrote the git history as well.